clients/opa/opa.go

```diff
@@ -12,26 +12,19 @@ import (
 	"path"
 	"strings"
 
-	"github.com/tidepool-org/go-common/clients/status"
+	"github.com/mdblp/go-common/clients/status"
 )
 
-// Client is the interface to opa.
-type Client interface {
+// API is the interface to opa.
+type API interface {
```

Comment:
Why changing the name of the type? It's called Client in all our services' clients...